tests/compose-tests/test-basic-unified.sh

```diff
@@ -6,6 +6,11 @@ dn=$(cd $(dirname $0) && pwd)
 . ${dn}/libcomposetest.sh
 
 prepare_compose_test "basic-unified"
+# Test --print-only, currently requires --repo
+rpm-ostree compose tree --repo=${repobuild} --print-only ${treefile} > treefile.json
```

Comment:
That should already work though since we're printing the version stderr, right?

Reply:
Yes, true.